docs/user/known-issues.md

```diff
@@ -0,0 +1,160 @@
+
+Flags:
+      --base-image string   name:tag of the base image to use for the build (default "kindest/base:v20181203-d055041")
+  -h, --help                help for node-image
+      --image string        name:tag of the resulting image to be built (default "kindest/node:latest")
+      --kube-root string    Path to the Kubernetes source directory (if empty, the path is autodetected)
+      --type string         build type, one of [bazel, docker, apt] (default "docker")
+
+Global Flags:
+      --loglevel string   logrus log level [panic, fatal, error, warning, info, debug] (default "warning")
+
+error building node image: failed to build kubernetes: failed to build images: exit status 2
+```
+
+Then you may try increasing the resource limits for the Docker engine on Mac.
+It is recommended that you have at least 4GB of RAM and disk space 
+(these estimates are for a single node kind cluster).
+
+Open the **Preferences** menu.
+
+<img src="./images/docker-pref-1.png"/>
+
+Go to the **Advanced** settings page, and change the settings there, see 
+[changing Docker's resource limits][Docker resource lims].
+<img src="./images/docker-pref-2.png"/>
+
+
+## Failure for cluster to properly start
+This issue is similar to a 
+[failure while building the node image](#failure-to-build-node-image).
+If the cluster creation process was successul but you are unable to see any
+Kubernetes resources running, for example:
+
+```
+$ docker ps
+CONTAINER ID        IMAGE                  COMMAND                  CREATED              STATUS              PORTS                      NAMES
+c0261f7512fd        kindest/node:v1.12.2   "/usr/local/bin/entr…"   About a minute ago   Up About a minute   0.0.0.0:64907->64907/tcp   kind-1-control-plane
+$ docker exec -it c0261f7512fd /bin/sh
+# docker ps -a
+CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
+#
+```
+
+or `kubectl` being unable to connect to the cluster,
+```
+$ export KUBECONFIG="$(kind get kubeconfig-path)"
+$ kubectl cluster-info
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+Unable to connect to the server: EOF
+```
+
+Then as in [kind#156][kind#156], you may solve this issue by claiming back some
+space on your machine by removing unused data or images left by the Docker 
+engine by running:
+```
+$ docker system prune
+```
+
+and/or:
+```
+$ docker image prune
+```
+
+You can verify the issue by exporting the logs (`kind export logs`) and looking
+at the kubelet logs, which may have something like the following:
+```
+Dec 07 00:37:53 kind-1-control-plane kubelet[688]: I1207 00:37:53.229561     688 eviction_manager.go:340] eviction manager: must evict pod(s) to reclaim ephemeral-storage
+Dec 07 00:37:53 kind-1-control-plane kubelet[688]: E1207 00:37:53.229638     688 eviction_manager.go:351] eviction manager: eviction thresholds have been met, but no pods are active to evict
+```
+
```

Comment:
In this section I would also add a reference to [the know issue fo Dind when using a btrfs partition.](https://github.com/kubernetes-sigs/kubeadm-dind-cluster/blob/master/README.md#requirements)

I had this issue when first started trying kind and was complicated to figure out because the symptoms where similar. In this case the logs for kubelet have something similar to this:
```
an 03 17:42:41 kind-1-control-plane kubelet[3804]: F0103 17:42:41.470269 3804 kubelet.go:1359] Failed to start ContainerManager failed to get rootfs info: failed to get device for dir "/
var/lib/kubelet": could not find device with major: 0, minor: 67 in cached partitions map
```

Reply:
Added! I was never clear on the btrfs problem. Thank you for bringing it up